exchanges/okx/okx.go

```diff
@@ -3348,7 +3348,7 @@ func (ok *Okx) GetBlockTrades(ctx context.Context, instrumentID string) ([]Block
 	}
 	params.Set("instId", instrumentID)
 	var resp []BlockTrade
-	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getBlockTradesEPL, http.MethodGet, common.EncodeURLValues(marketBlockTrades, params), nil, &resp, false)
+	return resp, ok.SendHTTPRequest(ctx, exchange.RestSpot, getBlockTradesEPL, http.MethodGet, common.EncodeURLValues(publicBlockTrades, params), nil, &resp, false)
```

Comment:
Nice work on the spot. Could you now update the response type to be in line with the docs?
https://www.okx.com/docs-v5/en/#block-trading-rest-api-get-block-trades

Reply:
Yep; done. Sorry for the delay. Had to fix PublicBlockTrades to test it efficiently.

Fixed 3fd398c